commented mon_party_sprites.asm

YamaArashi opened the pull request.

yenatch commented:
Maybe go with something generic like `GFX` instead of `TilePatterns`.

What does UnusedPartyMonSpriteFunction do?

YamaArashi commented:
OK. I changed the label names. I added a comment for UnusedPartyMonSpriteFunction. The function seems to be written incorrectly and doesn't do anything useful.
